packages/react/src/components/Breadcrumb/Breadcrumb.jsx

```diff
@@ -101,20 +104,22 @@ const Breadcrumb = ({ children, className, hasOverflow, ...other }) => {
         'breadcrumb--container__overflowfull': afterOverflowItems.length === 1,
       })}
       ref={breadcrumbRef}
-      data-testid="overflow"
```

Comment:
Looks like we are changing existing test id. Will need to add this to the issue to update in v3 but leave existing for now.

Reply:
reverted. added todo note